src/main/scala/com/datastax/sparkstress/ReadTask.scala

```diff
@@ -162,23 +132,26 @@ class FTSOneColumn(config: Config, ss: SparkSession) extends ReadTask(config, ss
  * Performs a full table scan but only retreives a single column from the underlying
  * table.
  */
-class FTSAllColumns(config: Config, ss: SparkSession) extends ReadTask(config, ss) {
-  def run(): Unit = {
-    val count = sc.cassandraTable[PerfRowClass](keyspace, table).count
+class FTSAllColumns(config: Config, ss: SparkSession) extends DatasetReadTask(config, ss) {
+  def run(): Unit = config.distributedDataType match {
+    case "rdd" => val count = sc.cassandraTable[PerfRowClass](keyspace, table).count
+    case _ => val count = read_columns(Seq("order_number", "qty", "color", "size", "order_time", "store"))
     println(s"Loaded $count rows")
   }
 }
 
 /**
  * Full Table Scan Five Columns
- * Performs a full table scan and only retreives 5 of the coulmns for each row
+ * Performs a full table scan and only retreives 5 of the columns for each row
  */
-class FTSFiveColumns(config: Config, ss: SparkSession) extends ReadTask(config, ss) {
-  def run(): Unit = {
-    val count = sc.cassandraTable[(UUID, Int, String, String, org.joda.time.DateTime)](keyspace,
-      table)
-      .select("order_number", "qty", "color", "size", "order_time")
-      .count
+class FTSFiveColumns(config: Config, ss: SparkSession) extends DatasetReadTask(config, ss) {
+  def run(): Unit = config.distributedDataType match {
+    case "rdd" =>
+      val count = sc.cassandraTable[(UUID, Int, String, String, org.joda.time.DateTime)](keyspace,
+        table)
+        .select("order_number", "qty", "color", "size", "order_time")
+        .count
+    case _ => val count = read_columns(Seq("order_number", "qty", "color", "size", "order_time"))
```

Comment:
Is having this println only in the second `case` intentional? Do we want to println in case `case "rdd" => ...` ?
This question applies to different FST.. classes as well.

Reply:
Good catch, this was not intentional. I'll change so this prints in all cases.